sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/PlanTest.scala

```diff
@@ -81,6 +81,8 @@ trait PlanTestBase extends PredicateHelper with SQLHelper { self: Suite =>
         ae.copy(resultId = ExprId(0))
       case lv: NamedLambdaVariable =>
         lv.copy(exprId = ExprId(0), value = null)
+      case pdf: PythonUDF =>
```

Comment:
Ur, `pdf`? Do we use like this before? Maybe, `udf: PythonUDF` is better?

Reply:
Yeah, a typo... :)